### DISCUSSION TITLE
Scrcpy for Android-x86

### DISCUSSION THREAD

**Discussion by toshiba6012** [Genymobile/feedback#17] [Genymotion Desktop] [answered]
Hello I really enjoy using Scrcpy on Linux I find it really productive. However I am a fan of Android-x86 and I am requesting a version for this platform. I use your app with the OTG wire and it runs flawless. I'm using Scrcpy for Android from this guy right here: https://gitlab.com/las2mile/scrcpy-android and it works better than a vncserver with no lag. Prime OS doesn't support a vncserver but this works for mirroring your computer screen to a tablet. But I want to mirror my phone to my Android-x86 computer so you would need to build a APK. I look forward to your response cheers.

Best regards
Ryan Johnson

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @toshiba6012 ,

Though it is hosted by us, we do not manage Scrcpy.
Please contact Scrcpy author and contributors at https://github.com/Genymobile/scrcpy

Thank you.
Best regards,
